test/app/database/models/numeric-id.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+
+const schema = mongoose.Schema({
+  _id: {
+    type: Number,
+    required: true
```

Comment:
Just out of curiosity, if `_id` isn't required here, does it effect the tests?

Also, in JSON API, a resource's id needs to be a string....so I'm not sure how we want to handle that. For now, it might make sense to only allow strings or ObjectIds.